include/psp2/fios2.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * \usergroup{SceFios2}
+ * \usage{psp2/fios2.h,SceFios2_stub}
+ */
+
+
+#ifndef _PSP2_FIOS2_H_
+#define _PSP2_FIOS2_H_
+
+#include <psp2/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SCE_FIOS2_OVERLAY_PATH_SIZE       (292)
+#define SCE_FIOS2_OVERLAY_PATH_MAX_LENGTH (SCE_FIOS2_OVERLAY_PATH_SIZE - 1)
+
+typedef int32_t SceFiosOverlayID;
+
+typedef enum SceFiosOverlayType {
+  // src replaces dst. All accesses to dst are redirected to src.
+  SCE_FIOS_OVERLAY_TYPE_OPAQUE      = 0,
+
+  // src merges with dst. Reads check src first, then dst. Writes go to dst.
+  SCE_FIOS_OVERLAY_TYPE_TRANSLUCENT = 1,
+
+  // src merges with dst. Reads check both src and dst, and use whichever has the most recent modification time.
+  // If both src and dst have the same modification time, dst is used.
+  // If no file exists at src or dst, dst is used; if no file exists at dst, but a file exists at src, src is used. Writes go to dst.
+  SCE_FIOS_OVERLAY_TYPE_NEWER       = 2,
+
+  // src merges with dst. Reads check src first, then dst. Writes go to src.
+  SCE_FIOS_OVERLAY_TYPE_WRITABLE    = 3
+} SceFiosOverlayType;
+  
+typedef struct SceFiosOverlay {
+  uint8_t type; // see SceFiosOverlayType
```

Comment:
Shouldn't these be using SceUint* prefix?

Reply:
it wasn't used in the kernel header, but ill add it anyway.